layouts/partials/tags.html

```diff
@@ -0,0 +1,7 @@
+<div class="tags">
+  {{ if .Params.tags }}
```

Comment:
Maybe this should wrap the `div` so that we don't even display the div when there are no tags.